src/python/pants/backend/python/subsystems/black.py

```diff
@@ -21,3 +21,6 @@ def register_options(cls, register):
             fingerprint=True,
             help="Path to formatting tool's config file",
         )
+        register(
+            "--options", advanced=True, type=str, fingerprint=True, help="Options to pass to black"
```

Comment:
I don't think we want to add this here. Were you only adding this so that `black.sh` has parity with `isort.sh`?

We should have a dedicated PR called something like "Add passthru args to Black goal".

Reply:
Yes, I was aiming for parity with the isort pre_commit. Specifically, I think it's potentially nice to inform and let the users run black themselves for the change: they feel in control. Would be open to discuss that behaviour.

Also makes sense to make it its own PR. I didn't manage to get the `-- --passthru --arguments` syntax to work, but if there is a way to do this, I would prefer that.

Reply:
See https://github.com/pantsbuild/pants/pull/8452 for that standalone PR (reduced the scope to only passing through the `--check` flag)